### DISCUSSION TITLE
Crop help

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#420] [Q&A] [answered]
How exactly do I use the crop function in scrcpy? I can't seem to figure it out.

**Comment by yume-chan** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1330321/172768420-0425e9ba-a710-470b-840c-a1092f44225b.png)

The syntax is same as the original Scrcpy (https://github.com/Genymobile/scrcpy#crop). Enter the width, height, horizontal offset, and vertical offset, separated by half-width colons (`:`), into the input box. 

For example, `120:120:40:40` means 120x120 pixels at 40, 40 offset.
